### PR TITLE
Add placeholder metadata page

### PR DESCRIPTION
**Pull request by Lholbourn**
This page is a placeholder for the future custom metadata page. It is linked to from the recordChecksResults page through the continue button to allow us to test this with E2E testing.

### CODE REVIEW

**Review by suzannehamilton — DISMISSED** [review]
Looks good to me, but I guess we shouldn't merge this until #246 is merged?

**Comment by Lholbourn** [comment]
> Looks good to me, but I guess we shouldn't merge this until #246 is merged?

Yeah I think it's best to wait since it relies on that work being in there :+1:

**Review by suzannehamilton — APPROVED** [review]

**Pull request merged**
